tests/cypress/integration/newfold-labs/wp-module-onboarding/phase-2-rudimentary.cy.js

```diff
@@ -0,0 +1,753 @@
+describe( 'Onboarding Phase 2 Flow', () => {
```

Comment:
Modified the chrome recordings to 
1. Set customer data.
2. Set Bluehost branding.
3. Wait for API calls and Live Preview Rendering to complete (the live preview rendering is flaky and the time to render changes, unfortunately Gutenberg/Editor has no way to determine whether a live preview has completed rendering or not)